### PR TITLE
Create the surveillance advertising page

### PR DESCRIPTION
**Pull request by davidfischer**
Build an overview of surveillance advertising and what people can do about it. I'd like to get some graphics here but I thought I'd get a first draft up.

Uses content from here: https://docs.readthedocs.io/en/stable/advertising/advertising-details.html

### CODE REVIEW

File: content/pages/learning-hub/surveillance-advertising.md [hunk 1]
```diff
@@ -1,6 +1,133 @@
 title: Surveillance Advertising
-slug: surveillance
-description: All about how you're being tracked around the web.
-subtitle: Good advertising doesn't stalk you.
+slug: surveillance-advertising
+description: Online advertising works by collecting data on millions of people without their consent to target ads better. You can opt-out and we can help!
+subtitle: Good advertising doesn't have to stalk you.
```
Comment (ericholscher) [inline content/pages/learning-hub/surveillance-advertising.md:4]: stalk seems a bit heavy here and insensitive -- maybe "follow you around the internet" or similar?

File: content/pages/learning-hub/surveillance-advertising.md [hunk 2]
```diff
@@ -1,6 +1,133 @@
...
 
-## TODO(ra): insert info about surveillance advertising
+
+Modern online advertising works by accumulating and processing data on millions of people
+usually without their consent to help target ads with ever more granularity.
+We call this **surveillance advertising** and believe in a different approach.
```
Comment (ericholscher) [inline content/pages/learning-hub/surveillance-advertising.md:9]: I'd probably bold the whole sentence, so people don't think we're in favor of surveillance ads:

```suggestion
**This is called surveillance advertising and believe in a different approach.**
```

File: content/pages/learning-hub/surveillance-advertising.md [hunk 3]
```diff
@@ -1,6 +1,133 @@
...
+such as the posts they've liked or the groups they are a member of.
+This data is very useful to target ads
+but all that data at least was given directly to Facebook willfully and voluntarily.
+Things get a little murkier when tracking users across the web on third-party sites.
+
+Building a profile from users' visits to third-party sites is typically accomplished by the use of trackers
+(sometimes called beacons or pixels).
+For example, if a site uses a tracker from an ad network and a user visits that site,
+the site operator can now target future advertising to that user – a known past visitor – with that network.
+This is called **retargeting** or **remarketing**.
```
Comment (ericholscher) [inline content/pages/learning-hub/surveillance-advertising.md:44]: Seems we're bolding all the stuff we don't like. I think we should probably give these italics, and highlight what _we believe in_.

File: content/pages/learning-hub/surveillance-advertising.md [hunk 4]
```diff
@@ -1,6 +1,133 @@
...
+Understandably, many people have concerns about these targeting techniques.
+If you've ever felt that ads "follow you around" while you browse the web, it's because of these.
+While the modern advertising industry has built enormous value,
+it's done so by centralizing massive amounts of data on as many people as possible.
+
+
+## We target our ads differently
+
+[Our vision]({filename}/pages/vision.md) for advertising is different.
+We target exclusively based on the content of the page and general geography (usually the country level).
```
Comment (ericholscher) [inline content/pages/learning-hub/surveillance-advertising.md:64]:
```suggestion
**We target exclusively based on the content of the page and general geography** (usually the country level).
```

**Review by ericholscher — APPROVED** [review]
This looks like a great start. I agree images would be good, but not a huge deal. I think the bigger thing is really trying to focus on the goal of this content. I think we want to highlight the negatives, but we also want to highlight the positives. It seems like the existing copy maybe focuses too much on the negative for my taste -- we should be highlighting what we're doing about it, not as much about the problem or definitions.

File: content/pages/learning-hub/surveillance-advertising.md (hunk 3 above)
Comment (davidfischer) [reply content/pages/learning-hub/surveillance-advertising.md:44]: That sounds good.

**Comment by davidfischer** [comment]
> the existing copy maybe focuses too much on the negative for my taste

I'll build out the section on contextual targeting a bit but this is definitely going to be a somewhat negative article. The article is about "surveillance advertising". Detailing why contextual targeting works is important but a major goal of this article is to get folks to opt-out.

**Review by ericholscher — COMMENTED** [review]
Looks good 👍

**Pull request merged**